zerver/lib/typed_endpoint.py

```diff
@@ -316,10 +316,12 @@ def parse_view_func_signature(
     "datetime_parsing": _("{var_name} is not a date"),
     "datetime_type": _("{var_name} is not a date"),
     "dict_type": _("{var_name} is not a dict"),
+    "enum": _("Invalid {var_name}"),
     "extra_forbidden": _('Argument "{argument}" at {var_name} is unexpected'),
     "float_parsing": _("{var_name} is not a float"),
     "float_type": _("{var_name} is not a float"),
     "greater_than": _("{var_name} is too small"),
+    "greater_than_equal": _("{var_name} is too small"),
```

Comment:
This error is raised currently when `waiting_period_threshold` setting is passed as negative. We previously raised "argument is negative" error, but `greater_than_equal` can be raised for other cases as well and thus updated the error to be as above.

Reply:
Sure, makes sense.